tests/test_external_tests.py

```diff
@@ -1,90 +1,121 @@
+import os
+import shlex
+import sys
 from pathlib import Path
 from tempfile import TemporaryDirectory
 
 import pytest
 
+import docker
 from dcqc import tests
-from dcqc.target import PairedTarget, SingleTarget
+from dcqc.target import SingleTarget
 from dcqc.tests import BaseTest, ExternalTestMixin, Process, TestStatus
 
 
-def test_that_the_file_extension_test_works_on_correct_files(test_targets):
-    target = test_targets["good"]
-    test = tests.FileExtensionTest(target)
-    test_status = test.get_status()
-    assert test_status == TestStatus.PASS
-
-
-def test_that_the_file_extension_test_works_on_correct_remote_file(test_targets):
-    target = test_targets["remote"]
-    test = tests.FileExtensionTest(target)
-    test_status = test.get_status()
-    assert test_status == TestStatus.PASS
-
-
-def test_that_a_tiff_file_with_good_extensions_is_passed(test_targets):
-    target = test_targets["tiff"]
-    test = tests.FileExtensionTest(target)
-    assert test.get_status() == TestStatus.PASS
-
+def test_that_all_external_tests_inherit_from_the_mixin_first():
+    tests = BaseTest.list_subclasses()
```

Comment:
Code smell:
`Redefining name 'tests' from outer scope (line 10) PylintW0621:redefined-outer-name`

Reply:
addressed in latest commit